mlrun/runtimes/daskjob.py

```diff
@@ -603,9 +603,9 @@ def enrich_dask_cluster(
         env.append(spec.extra_pip)
 
     pod_labels = get_resource_labels(function, scrape_metrics=config.scrape_metrics)
-    # TODO: 'dask-worker' has deprecation notice, user 'dask worker' instead
+    # TODO: 'dask-worker' has deprecation notice, use 'dask worker' instead
```

Comment:
unrelated to pr, why not doing it then?

Reply:
looks like the new cli was introduced in [2022.10.0](https://docs.dask.org/en/stable/changelog.html#v2022-10-0)
we can't upgrade before we drop python 3.7 support - fixing the comment

Reply:
glad you tested and updated comment. now it is meaningful and actionable 🚀